src/components/formFields/checkbox/Checkbox.module.css

```diff
@@ -1,7 +1,206 @@
+  vertical-align: middle;
+  margin-right: 3px;
+  width: 13px;
+  height: 13px;
+  background-color: var(--conduction-input-checkbox-hover-color);
+  border-width: var(--conduction-input-checkbox-hover-border-width, var(--conduction-input-checkbox-border-width));
+  border-style: var(--conduction-input-checkbox-hover-border-style, var(--conduction-input-checkbox-border-style));
+  border-color: var(--conduction-input-checkbox-hover-border-color, var(--conduction-input-checkbox-border-color));
+  border-radius: var(--conduction-input-checkbox-hover-border-radius, var(--conduction-input-checkbox-border-radius));
+  box-shadow: var(--conduction-input-checkbox-hover-box-shadow, var(--conduction-input-checkbox-box-shadow));
+}
+
+/* Checked */
+
+.checkbox:checked + label::before {
+  content: " ";
+  display: inline-block;
+  vertical-align: middle;
+  margin-right: 3px;
+  width: 13px;
+  height: 13px;
+  background-color: var(--conduction-input-checkbox-checked-color);
+  border-width: var(--conduction-input-checkbox-checked-border-width);
+  border-style: var(--conduction-input-checkbox-checked-border-style);
+  border-color: var(--conduction-input-checkbox-checked-border-color);
+  border-radius: var(--conduction-input-checkbox-checked-border-radius);
+  box-shadow: var(--conduction-input-checkbox-checked-box-shadow);
+}
+
+.checkbox:checked:hover + label::before {
+  content: " ";
+  display: inline-block;
+  vertical-align: middle;
+  margin-right: 3px;
+  width: 13px;
+  height: 13px;
+  background-color: var(--conduction-input-checkbox-checked-hover-color, var(--conduction-input-checkbox-checked-color));
+  border-width: var(--conduction-input-checkbox-checked-hover-border-width, var(--conduction-input-checkbox-checked-border-width));
+  border-style: var(--conduction-input-checkbox-checked-hover-border-style, var(--conduction-input-checkbox-checked-border-style));
+  border-color: var(--conduction-input-checkbox-checked-hover-border-color, var(--conduction-input-checkbox-checked-border-color));
+  border-radius: var(--conduction-input-checkbox-checked-hover-border-radius, var(--conduction-input-checkbox-checked-border-radius));
+  box-shadow: var(--conduction-input-checkbox-checked-hover-box-shadow, var(--conduction-input-checkbox-checked-box-shadow));
+}
+
+.checkbox:checked + label::after {
+  content: " ";
+  background-image: url("data:image/svg+xml,%3Csvg id='eXTGSQIndRe1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' viewBox='0 0 13 13' shape-rendering='geometricPrecision' text-rendering='geometricPrecision'%3E%3Cg transform='translate(-85.642166-156.631362)'%3E%3Cpath d='M86.73927,163.90021l1.92835-2.05763l4.165103,4.176559c-.692268.524-1.562158,1.361078-1.562158,2.198156L86.73927,163.90021Z' fill='%23fff' stroke-width='0'/%3E%3Cpath d='M95.2777,158.04542l2.26736,1.67416-6.27449,8.49772-1.917847-1.798161L95.2777,158.04542Z' fill='%23fff' stroke-width='0'/%3E%3C/g%3E%3C/svg%3E%0A");

```

Comment:
This might be better as a design token, I'd imagine not all themes wanting this specific checkmark.